static/app/components/breadcrumbs.tsx

```diff
@@ -102,8 +103,19 @@ const BreadcrumbList = styled('div')`
   padding: ${space(1)} 0;
 `;
 
-const BreadcrumbLink = styled(({preserveGlobalSelection, ...props}) =>
-  preserveGlobalSelection ? <GlobalSelectionLink {...props} /> : <Link {...props} />
+type BreadcrumbLinkProps = {
+  children?: React.ReactNode;
```

Comment:
can we move the optional props to the end of this object?

Reply:
sure, will follow up